lib/model/autocomplete.dart

```diff
@@ -615,6 +602,59 @@ class MentionAutocompleteView extends AutocompleteView<MentionAutocompleteQuery,
     return userAName.compareTo(userBName); // TODO(i18n): add locale-aware sorting
   }
 
+  void computeWildcardMentionResults({
```

Comment:
nit in commit message:

    autocomplete: Support @-wildcard in user-mention autocomplete
    
    The implementation logic is similar to the zulip-mobile implementation:
      https://github.com/zulip/zulip-mobile/blob/a115df1f71c9dc31e9b41060a8d57b51c
017d786/src/autocomplete/WildcardMentionItem.js

There's a few other things going on in that file, so this link can be made more precise with a line-number range:
https://github.com/zulip/zulip-mobile/blob/a115df1f71c9dc31e9b41060a8d57b51c017d786/src/autocomplete/WildcardMentionItem.js#L92-L140

Reply:
Then there's a few ways we've made this differ from zulip-mobile, so let's briefly mention those:
* suppress wildcards on silent mentions
* match "stream" queries even on new servers (but emit `@**channel**`)
* possibly something I'm forgetting